site/docs/README.md

```diff
@@ -93,3 +93,7 @@ Works! :tada:
 (Last highlight: Translated Bot Descriptions)
 
 </div>
+
+<ClientOnly>
+  <LanguagePopup />
+</ClientOnly>
```

Comment:
Is it possible to use the [layout slots](https://v2.vuepress.vuejs.org/reference/default-theme/extending.html#layout-slots) instead of manually adding components to each page?

Reply:
Why not?